### DISCUSSION TITLE
Can't install Tomcat on Ubuntu EC2 instance (AWS). Help needed!

### DISCUSSION THREAD

**Discussion by ajanidev** [community/community#30294] [Codespaces]
This is the script I added to the user data:

```
#!/bin/bash
sudo apt update
sudo apt upgrade -y
sudo apt install openjdk-11-jdk -y
sudo apt install tomcat10 tomcat10-admin tomcat10-docs tomcat10-common git -y
```
It's an Ubuntu 22.04 AMI on AWS and it returns the following code after I SSH and do a systemctl status:

`Failed to start tomcat.service: Unit tomcat.service not found.`

Should have been a pretty straight forward thing. I've tried a lot of things after doing a lot of googling and youtubing and nothing seems to work. Any idea what I may have done wrong?

**Comment by bdmac** [comment]
I don't know much about Tomcat but found https://askubuntu.com/questions/1255747/failed-to-start-tomcat-unit-tomcat-service-not-found and I'm wondering if the `tomcat.service` file was created and if so I wonder if it's in a different place than usual and something needs to be pointed to it?
